docs/design/windows-design.md

```diff
@@ -99,10 +100,12 @@ port (provided with `local_ip` option). This local address is the one configured
 ### OVS bridge interface configuration
 
 Since OVS is also responsible for taking charge of the network of the host, an interface for the OVS bridge
-is required on which the host network settings are configured. It is created and enabled when creating
-the OVS bridge, and the MAC address should be changed to be the same as the uplink interface. Then the IP
-address and the route entries originally configured on the uplink interface should also be migrated to
-the interface.
+is required on which the host network settings are configured. The virtual network adapter which is created
```

Comment:
Could we reflect this in the hns_integration.svg diagram?

Reply:
updated.

Reply:
Do you think we have some way to indicate the br-int interface is a virtual network adapter in the diagram?